configure.ac

```diff
@@ -60,11 +62,13 @@ AC_SUBST(top_top_srcdir)
 MAJOR_VERSION=ucx_ver_major
 MINOR_VERSION=ucx_ver_minor
 PATCH_VERSION=ucx_ver_patch
-VERSION=$MAJOR_VERSION.$MINOR_VERSION.$PATCH_VERSION
+EXTRA_VERSION=ucx_ver_extra
+VERSION=$MAJOR_VERSION.$MINOR_VERSION.$PATCH_VERSION$EXTRA_VERSION
 SOVERSION=libucx_so_version
 AC_SUBST(MAJOR_VERSION)
 AC_SUBST(MINOR_VERSION)
 AC_SUBST(PATCH_VERSION)
+AC_SUBST(EXTRA_VERSION)
```

Comment:
What is extra version is ? In openshmem we used something call vendor string which let's vendors put whatever they want.

Reply:
it could be a vendor string, but also something like "rc1" (though we don't require it today)